### PR TITLE
Adding data model classes

### LINKED ISSUE #21: Add data model classes
-Add only the data model classes  without functions and tests
-Run `pipenv run python manage.py makemigrations projboard` command  to create the database table

### PR DESCRIPTION
**Pull request by Shalevro2**
Updating models.py- creating the data model classes

Running `pipenv run python manage.py makemigrations projboard` command to create the database table

**Note**: Most changes in migrations directory had been auto-generated.

Resolve #21

### CODE REVIEW

**Review by Anas-mhameed — APPROVED** [review]
LGTM

File: projboard/models.py [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+
+    user_id- FK to User model
+    title- A string, the title of article
+    subject_id- FK to Subject model
+    content- A string, the content of article
+    date- A DateTime object, the date the article created
+    """
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=1000)
+    subject_id = models.ForeignKey(Subject, on_delete=models.SET_NULL, null=True)
```
Comment (Anas-mhameed) [inline projboard/models.py]: You should add , blank = True in order to make this field not required to allow null value

**Review by Anas-mhameed — CHANGES_REQUESTED** [review]

File: projboard/models.py (hunk 1 above)
Comment (Anas-mhameed) [reply projboard/models.py]: in subject_id

Comment (Shalevro2) [reply projboard/models.py]: Fixed, look [here](https://github.com/redhat-beyond/Smarticle/blob/77494c0919ecb7ff5493139a518031e888ae5909/projboard/models.py)

**Review by rawadabu — APPROVED** [review]
LGTM

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by yaelnoyman1 — APPROVED** [review]
LGTM

**Review by Yarboa — CHANGES_REQUESTED** [review]
I think data population is missing,
pytest framework does not exist yet, at lease use migration tests.
please refer 
https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/migrations/0004_messages_test_data.py

**Comment by Shalevro2** [comment]
> I think data population is missing, pytest framework does not exist yet, at lease use migration tests. please refer https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/migrations/0004_messages_test_data.py

Hi @Yarboa ,

I created a test data class for each model.
Please take a look [here](https://github.com/redhat-beyond/Smarticle/pull/22/commits/94272beb5cfd6357cadb0f34a6ea5ea096c79a2c)

**Review by Yarboa — APPROVED** [review]

**Pull request merged**